frontend/src/services/graphql/ShareObject/index.js

```diff
@@ -1,11 +1,2 @@
-export * from './addSharedItem';
-export * from './approveShareObject';
 export * from './createShareObject';
```

Comment:
`createShareobject` is placed in the global `services` because it is used in the Catalog module right? (https://github.com/awslabs/aws-dataall/pull/581/files#diff-d9b39fcf99c580c2d427aefaf3a10d31389835997ae58d4380d5755e87c37b97) 
Wouldn't it be better to move the `RequestAccessModal` inside the `Shares` module? I know that it was placed with the Catalog before but if I am a developer working on Shares the first place that I am going to look at is the `Shares` module, what do you think?

Reply:
I believe `RequestAccessModal` is being used by Catalog to display search results using access, resourcetypes and some other properties, so it is directly related and being used only by Catalog, therefore it should be in the same module.

Reply:
And going by the functionality of the `createShareObject` service, although it's currently used by the Catalog module, it can also be used by other modules to share an item. That is why it is placed in the global services folder